product_docs/docs/pem/9/registering_database_server.mdx

```diff
@@ -56,6 +56,7 @@ Append command line options to the command string when invoking the pemworker ut
 | Option&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
 | ----------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | `--pem-user`                        | Specifies the name of the PEM administrative user (must have the pem_admin role) on the PEM server to use to write the server details to the PEM database. Required.                                                                                                                                                                                                                              |
+| `--pem-port`                        | Specifies the port on which to connect to the PEM server when writing the server details to the PEM database. Optional, defaults to the value of `--pem-port` set during agent registration (PEM 9.8 or newer, only). Otherwise defaults to 5432.                                                                                                                                                                                                                              |
```

Comment:
```suggestion
| `--pem-port`                        | Specifies the port on which to connect to the PEM server when writing the server details to the PEM database. Optional, defaults to the value of `--pem-port` set during agent registration (PEM 9.8 or newer only). Otherwise defaults to 5432.                                                                                                                                                                                                                              |
```